src/faebryk/libs/exceptions.py

```diff
@@ -162,7 +165,10 @@ def raise_errors(self):
                 ):
                     displayed_errors.append(error)
 
-            raise ExceptionGroup(self.group_message, displayed_errors)
+            if len(displayed_errors) > 1:
```

Comment:
This improves debugging behaviour in the case of a single error killing the build